package.json

```diff
@@ -13,6 +13,8 @@
     "url": "https://github.com/taskcluster/taskcluster-queue.git"
   },
   "dependencies": {
+    "ajv": "^4.2.0",
+    "ajv-keywords": "^1.1.1",
```

Comment:
@jonasfj So, `taskcluster-lib-api` wants `^3.8.5`. That happens to be the highest level in the tree (`taskcluster-lib-api` is queue's dependency). So, `ajv` is installed in queue's `node_modules`, and the version is 3.8.10. Now, the table has its own `ajv`, which is 4.8.2. It doesn't have its own `ajv-keywords`, however. As per default npm 3's behavior, `ajv-keywords` goes to the queue's modules. What seems to be happening, is that `ajv-keywords` uses the `ajv` in the same directory as it self, not the one in the `table`'s npm_modules. It probably happens because `ajv` is a peer dependency, not regular dependency. (I assume)

We could probably use a newer `ajv` in `tc-lib-api`, that could be a solution.